content/2024-04-15-t8code_version200_released.md

```diff
@@ -0,0 +1,28 @@
+Title: t8code-2.0.0 released 
+Date: 2024-04-15 14:00 
+Category: News 
+Authors: Jakob Fußbroich
```

Comment:
```suggestion
Authors: Jakob Fußbroich
Image: release200/Release_2_0.png
```
Please use the image keyword the website implementation provides: https://github.com/DLR-AMR/t8code-website/pull/70